src/types.d.ts

```diff
@@ -23,6 +23,10 @@ export interface Device {
   sim_id: string
 }
 
+export interface DeviceWithFetchedAt extends Device {
```

Comment:
why not just add to Device?

Reply:
to keep the api's `DeviceType` distinct from the representation in the app

Reply:
i don't think that's too important, maybe just add an inline comment that that's not populated by the api?

Reply:
cool, that works